http/src/main/resources/reference.conf

```diff
@@ -23,11 +23,15 @@ dataproc {
   #vpcSubnet = "default"
   # This image is used for legacy jupyter image where hail is compatible with earlier version of spark
   legacyCustomDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-1-2-79-debian9-9ba5905"
-  customDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-1-4-15-debian9-9ba5905"
+  customDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-dataproc-1-4-15-debian9-hhhhhhh"
 
   dataprocReservedMemory = 6g
 }
 
+gce {
```

Comment:
I added a `gce` section to config in https://github.com/DataBiosphere/leonardo/pull/1281/ as well, we'll have to merge the two

Reply:
👍 will rebase